core/src/main/scala/org/apache/spark/h2o/backends/external/RestCommunication.scala

```diff
@@ -218,6 +194,31 @@ trait RestCommunication extends Logging {
   }
 }
 
+object RestCommunication {
```

Comment:
What is the reason of moving the `decodeSimpleParam`, `decodeParams` and `decodeArray` to an object?

Reply:
Originally I wanted to reuse these methods outside this class to encode array I need to pass to params, but at the end I have found a cleaner approach. Thanks!